chef/cookbooks/crowbar/recipes/default.rb

```diff
@@ -272,8 +272,15 @@
   realm = nil
 end
 
-template "/opt/dell/crowbar_framework/rainbows.cfg" do
-  source "rainbows.cfg.erb"
+template "/etc/sysconfig/crowbar" do
+  source "sysconfig.crowbar.erb"
+  owner "crowbar"
+  group "crowbar"
+  mode "0644"
+end
+
+template "/opt/dell/crowbar_framework/config/puma.rb" do
+  source "puma.rb.erb"
   owner "crowbar"
   group "crowbar"
```

Comment:
Can you change the ownership to root:root? It makes no sense to have this owned by the crowbar user/group.